src/sp/it/pl/util/action/Action.java

```diff
@@ -439,7 +440,7 @@ public String toString() {
 	@IsConfig(name = "Media shortcuts supported", editable = EditMode.NONE, info = "Whether media shortcuts are supported on this system")
 	private static final boolean isMedialShortcutsSupported = true;
 	private static boolean isRunning = false;
-	private static Hotkeys hotkeys = new Hotkeys(Platform::runLater);
+	private static Hotkeys hotkeys = isGlobalShortcutsSupported() ? new Hotkeys(Platform::runLater) : null;
```

Comment:
Makes the code more complex and will make the work more difficult when converting to Kotlin. Removed.